check_breaking_individual.bash

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+APIS="users ad settings friends session config inventory presence notification rank custom"
```

Comment:
Does this need `analytics-api` as well?

Reply:
Nope, analytics is an internal-only API for the developer-API to use.